samples/boards/ti/cc13x2_cc26x2/system_off/src/main.c

```diff
@@ -28,26 +25,24 @@ extern void CC1352R1_LAUNCHXL_shutDownExtFlash(void);
 void main(void)
 {
 	uint32_t config, status;
-	const struct device *gpiob;
 
 	printk("\n%s system off demo\n", CONFIG_BOARD);
 
 	/* Shut off external flash to save power */
 	CC1352R1_LAUNCHXL_shutDownExtFlash();
 
 	/* Configure to generate PORT event (wakeup) on button 1 press. */
-	gpiob = device_get_binding(PORT);
-	if (!gpiob) {
-		printk("error\n");
+	if (!device_is_ready(sw0_gpio.port)) {
+		printk("%s: device not ready.\n", sw0_gpio.port->name);
 		return;
 	}
 
-	gpio_pin_configure(gpiob, PIN, GPIO_INPUT | PULL_UP);
+	gpio_pin_configure_dt(&sw0_gpio, GPIO_INPUT);
 
 	/* Set wakeup bits for button gpio */
-	config = IOCPortConfigureGet(PIN);
+	config = IOCPortConfigureGet(sw0_gpio.pin);
 	config |= IOC_WAKE_ON_LOW;
-	IOCPortConfigureSet(PIN, IOC_PORT_GPIO, config);
+	IOCPortConfigureSet(sw0_gpio.pin, IOC_PORT_GPIO, config);
```

Comment:
for future: should use standard gpio API (if wake flag is not supported it can be added now that gpios support vendor flags)